addons/survey/models/survey_question.py

```diff
@@ -219,6 +230,14 @@ def _compute_background_image_url(self):
             else:
                 question.background_image_url = question.survey_id.background_image_url
 
+    @api.depends('question_type')
+    def _compute_answer_type(self):
```

Comment:
Well looking at second commit, indeed we can keep "value_scale" as all other values are inlined with question type, you are right :)